programs/bpf_loader/src/syscalls/mod.rs

```diff
@@ -4261,6 +4262,160 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_syscall_get_epoch_stake() {
+        let config = Config::default();
+        let compute_budget = ComputeBudget::default();
+        let sysvar_cache = Arc::<SysvarCache>::default();
+
+        let expected_epoch_stake = 55_000_000_000u64;
+        let expected_cus = compute_budget.syscall_base_cost
+            + 32 / compute_budget.cpi_bytes_per_unit
```

Comment:
Would be nice if we didn't have to use this magic number `32` (and looking back, it would have been better to overtly define it as the length of the (pubkey-bytes) data input in the SIMD). Can you use `PUBKEY_BYTES` here to be clear?